frontend/content/locale/en.json

```diff
@@ -1,4 +1,20 @@
 {
+  "featuresHeading": "Engineered for student success",
+  "feature1": "Well-designed",
+  "featureMsg1": "Designed to be visually appealing with usability in mind",
+  "feature2": "Moderation",
+  "featureMsg2":
+    "Built with all the tools you need to ensure responsible usage",
+  "feature3": "UofT Support",
+  "featureMsg3":
+    "Integrated with Quercus & Shibboleth for a seamless instructor experience",
+  "feature4": "Accessible",
+  "featureMsg4":
+    "Follows WAI-ARIA principles, enables students with disabilties",
+  "feature5": "LaTeX/Markdown",
+  "featureMsg5": "Communicating math concepts is hard without symbols!",
+  "feature6": "Open Source",
+  "featureMsg6": "Completely transparent & open to contribution from all!",
   "heroHeading": "Communicate with students online, <span>seamlessly</span>",
```

Comment:
Suggestion: can we add these keys to es.json and fr.json, no need to translate it! a simple `ES-featureMsg6` works! It's to make sure the server doesn't scream at us for missing translations